scripts/rewrite-decl-statements-for-closure-test.js

```diff
@@ -95,14 +95,47 @@ function transform(srcFile, rootDir) {
   });
 
   traverse(ast, {
-    'ImportDeclaration|ExportNamedDeclaration'({node}) {
-      if (node.source) {
-        rewriteDeclarationSource(node, srcFile, rootDir);
+    ExportNamedDeclaration(path) {
```

Comment:
Given that we're adding a bunch of stuff here, should we add to the docs above in this file?

Also, should we add notes about the new requirements to our docs about writing closure-friendly code?

Reply:
Added some docs

Reply:
I see the docs added in markdown, but should there be anything in a block comment here / above summarizing what we're doing?

Reply:
woops...forgot to push that change